tests/test_alerts.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+
+
+@pytest.fixture
+def browser():
+    driver = webdriver.Chrome()
+    # driver = webdriver.Firefox()
+    driver.implicitly_wait(5)
+    driver.get("https://demo.seleniumeasy.com/javascript-alert-box-demo.html")
+    yield driver
+    driver.quit()
+
+
+def test_java_script_alert_box_text(browser):
+    click_me_button = browser.find_element(By.CSS_SELECTOR, "div.col-md-6.text-left div.panel.panel-primary:first-of-type button")
```

Comment:
`    click_me_button = browser.find_element(By.CSS_SELECTOR, "div.col-md-6.text-left div.panel.panel-primary:first-of-type button")
    click_me_button.click()`
    
    you can just write:
    browser.find_element(*click_me_button).click() , click_me_button var defined above

Reply:
ok, it is done